AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/StorageEngineAdapter+SQLite.swift

```diff
@@ -27,6 +27,9 @@ final class SQLiteStorageEngineAdapter: StorageEngineAdapter {
     // our system and find an optimal value.
     static var maxNumberOfPredicates: Int = 950
 
+    // Create SQLite indexes when SQLite tables are created
+    var shouldCreateSQLiteIndexes: Bool = true
```

Comment:
do we still need this or we just used this while testing the feature?

Reply:
Yup, this is needed for the tests.